x-pack/plugins/security_solution/package.json

```diff
@@ -10,7 +10,7 @@
     "cypress": "../../../node_modules/.bin/cypress",
     "cypress:open": "yarn cypress open --config-file ./cypress/cypress.json",
     "cypress:open:ccs": "yarn cypress:open --config integrationFolder=./cypress/ccs_integration",
-    "cypress:open-as-ci": "node ../../../scripts/functional_tests --config ../../test/security_solution_cypress/visual_config.ts",
+    "cypress:open-as-ci": "TZ=UTC node ../../../scripts/functional_tests --config ../../test/security_solution_cypress/visual_config.ts",
```

Comment:
The tests were failing because they expected to be in the UTC timezone rather than the timezone of the person running the test. This was the only way I found to set it (tried adding to cypress config), but that didn't work

Reply:
cc @MadameSheema Have you seen this issue before?